pkg/costmodel/costmodel.go

```diff
@@ -51,6 +52,7 @@ const (
 	remoteEnabled  = "REMOTE_WRITE_ENABLED"
 	thanosEnabled  = "THANOS_ENABLED"
 	thanosQueryUrl = "THANOS_QUERY_URL"
+	thanosOffset   = "THANOS_QUERY_OFFSET"
```

Comment:
Is this being used?

Reply:
Ah, crap - looks like all of these got added back.